src/azul/indexer/document.py

```diff
@@ -396,8 +400,19 @@ def translate_fields(cls,
                 return [cls.translate_fields(val, field_types, forward=forward, path=path) for val in doc]
             else:
                 assert False, (path, type(doc))
-        elif isinstance(field_types, FieldType):
-            field_type = field_types
+        elif isinstance(field_types, (list, FieldType)):
+            if isinstance(field_types, list):
+                field_type = one(field_types)
+                assert isinstance(field_type, FieldType)
+
+                # In english: if forward and doc is not None, then if
```

Comment:
Not only is "In english" obvious, so is what follows it. Document the why, not the how.